release-notes/4.0/4.0.0-preview3.md

```diff
@@ -0,0 +1,121 @@
+# Release Notes
+
+## Microsoft.Data.SqlClient 4.0.0-preview3.x.x released 20 October 2021
+
+This update brings the below changes over the previous release:
+
+### Breaking changes over preview release v4.0.0-preview2
+
+- Dropped support for .NET Core 2.1 [#1272](https://github.com/dotnet/SqlClient/pull/1272)
+- Exception will not be thrown if a User ID is provided in the connection string when using `ActiveDirectoryIntegrated` authentication [#1359](https://github.com/dotnet/SqlClient/pull/1359)
+
+### Added
+
+- Add `GetFieldValueAsync<T>` and `GetFieldValue<T>`  support for `XmlReader`, `TextReader`, `Stream` [#1019](https://github.com/dotnet/SqlClient/pull/1019)
```

Comment:
Could you refer to the [related topic below](https://github.com/dotnet/SqlClient/pull/1363/files#diff-148a966e6ab5b4bdd75f79a5fe4b4f743917096acf3ce84f9427a26792e5e76cR36) with a "read more" tag?